tests/cypress/tests-e2e/featureSelection.cy.js

```diff
@@ -445,5 +445,84 @@ describe('Testing the feature selection', () => {
+            // Import KML file
+            const fileName = 'external-kml-file.kml'
+            const localKmlFile = `tests/cypress/fixtures/import-tool/${fileName}`
+            cy.goToMapView(
+                {
+                    layers: 'test.wms.layer',
+                },
+                true
+            )
+            cy.wait(['@routeChange', '@layers', '@topics', '@topic-ech'])
+
+            cy.openMenuIfMobile()
+            cy.get('[data-cy="menu-tray-tool-section"]:visible').click()
+            cy.get('[data-cy="menu-advanced-tools-import-file"]:visible').click()
+            cy.get('[data-cy="import-file-local-btn"]:visible').click()
+            cy.readFile(localKmlFile, null).as('printKmlFixture') // .readFile instead of .fixture because .fixture caches the file and makes it impossible to re-read it in another test
+            cy.get('[data-cy="file-input"]').selectFile('@printKmlFixture', {
+                force: true,
+            })
+            cy.get('[data-cy="import-file-load-button"]:visible').click()
+            cy.get('[data-cy="file-input-text"]').should('contain.value', fileName)
+
+            cy.wait(['@icon-sets', '@icon-set-babs', '@icon-set-default'])
+
+            cy.get('[data-cy="import-file-close-button"]:visible').realClick()
+            cy.readStoreValue('state.layers.activeLayers.length').should('eq', 2)
+            cy.readStoreValue('getters.visibleLayers.length').should('eq', 2)
+
+            cy.closeMenuIfMobile()
+
+            cy.checkOlLayer([
+                'test.background.layer2',
+                { id: 'test.wms.layer', opacity: 0.75 },
+                fileName,
+            ])
+
+            cy.get('[data-cy="ol-map"]').as('olMap').should('be.visible')
+            cy.log(
+                'Selecting a rectangle (by click&drag) while pressing SHIFT, should start a rectangle identification of features'
+            )
+            drawRectangleOnMap({
+                x: 10,
+                y: 10,
+            })
+
+            cy.log('making sure 51 items are requested when selecting a dragbox on the map') // including the one from the kml file
+            cy.wait('@identify')
+                .its('request.query.limit')
+                .should('eq', `${DEFAULT_FEATURE_COUNT_RECTANGLE_SELECTION}`)
+            // waiting for each feature detail to be loaded (can take a while with the stubbing, so it can lead to timeouts
+            // with further selectors if not properly waited)
+            for (
+                let featureCount = 0;
+                featureCount < DEFAULT_FEATURE_COUNT_RECTANGLE_SELECTION;
+                featureCount++
+            ) {
+                cy.wait(`@htmlPopup`)
+            }
+            cy.window().then((win) => {
+                cy.stub(win, 'print').as('print')
+            })
+            cy.get('[data-cy=print-button]').should('be.visible').click()
+            cy.window().then((_) => {
```

Comment:
what is the purpose of calling `.window()` without using the result of the call?